packages/syft/setup.cfg

```diff
@@ -100,11 +100,11 @@ dev =
     safety>=2.4.0b2
 
 telemetry =
-    opentelemetry-api==1.14.0
-    opentelemetry-sdk==1.14.0
-    opentelemetry-exporter-jaeger==1.14.0
-    opentelemetry-instrumentation==0.35b0
-    opentelemetry-instrumentation-requests==0.35b0
+    opentelemetry-api==1.24.0
```

Comment:
These code paths are not enabled so this could easily break the instrumentation we have, I think for now ill remove these and focus on the main package changes first.